io.catenax.parts_analyses/1.0.0/metadata.json

```diff
@@ -1 +1 @@
-{ "status" : "release"}
\ No newline at end of file
+{ "status" : "deprecated"}
```

Comment:
@bs-jokri is it deprecated or deprecate?

Reply:
I now changed the information in metadata file to deprecate